java/com/google/cose/utils/Headers.java

```diff
@@ -0,0 +1,48 @@
+package com.google.cose.utils;
+
+/**
+ * This class contains all the header keys that we are going to be using in COSE implementation.
+ */
+public class Headers {
```

Comment:
Nit: these aren't just header keys. These are simply COSE ids defined by IANA for various purposes.